pytorch_blade/benchmark/TorchBench/test_torch_bench.sh

```diff
@@ -85,7 +85,11 @@ do
 done
 
 # performance anaysis
-python3 results_anaysis.py -t ${results} -i ${oss_dir} -p ${RELATED_DIFF_PERCENT}
+if [ $1 == "cuda" ]; then
+  python3 results_anaysis.py -t ${results} -i ${oss_dir} -p ${RELATED_DIFF_PERCENT} -f "cuda"
+else
+  python3 results_anaysis.py -t ${results} -i ${oss_dir} -p ${RELATED_DIFF_PERCENT} -f "cpu"
```

Comment:
nit: just use `-f $1` ?

Reply:
$1 could be "cuda", "cpu" or "aarch64", so I guess we cannot just use `-f $1`